steps/autify-test-run/1.0.0/step.yml

```diff
@@ -0,0 +1,84 @@
+title: Autify test run
+summary: |
+  Run a test plan on Autify for Mobile.
+description: |
+  [Autify for Mobile](https://autify.com/mobile) is a test automation platform for your native app.
+  This step allows you to upload your app built in the previous steps to Autify and start a pre-recorded test plan against the build.
+
+  Optionally, this step can wait until the test plan finishes and fail if it doesn't pass.
+
+  Also, you can specify already uploaded build and just start a test plan against it.
+
+  If you want to upload your app to Autify, you need to build your app first by `Android Build`, `Xcode build for simulator`, etc.
+  and refer the target build path (*.apk or *.app) at `Build path to test` input.
+website: https://github.com/autifyhq/bitrise-step-autify-test-run
+source_code_url: https://github.com/autifyhq/bitrise-step-autify-test-run
+support_url: https://github.com/autifyhq/bitrise-step-autify-test-run/issues
+published_at: 2022-08-30T13:53:16.808596-07:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-test-run.git
+  commit: 65596a469477a82f51980165e76d1891304ff610
+project_type_tags:
+- ios
+- android
+- xamarin
```

Comment:
[xamarin is not officially supported on Bitrise](https://blog.bitrise.io/post/xamarin-support-ends-in-2022-on-bitrise), pls remove this project type tag.